apis/apiextensions/v1beta1/composition_types.go

```diff
@@ -144,11 +145,11 @@ type Patch struct {
 	// Type sets the patching behaviour to be used. Each patch type may require
 	// its' own fields to be set on the Patch object.
 	// +optional
-	// +kubebuilder:validation:Enum=FromCompositeFieldPath;PatchSet
+	// +kubebuilder:validation:Enum=FromCompositeFieldPath;PatchSet;ToCompositeFieldPath
 	// +kubebuilder:default=FromCompositeFieldPath
 	Type PatchType `json:"type,omitempty"`
 
-	// FromFieldPath is the path of the field on the upstream resource whose value
+	// FromFieldPath is the path of the field on the composed resource whose value
 	// to be used as input. Required when type is FromCompositeFieldPath.
 	// +optional
```

Comment:
`FromFieldPath` and `ToFieldPath` are used bi-directionally so could each refer to a field on either the composite or composed resource.

```suggestion
	// FromFieldPath is the path of the source field on the composed or 
	// composite resource whose value is to be used as input. Required 
	// when type is FromCompositeFieldPath or ToCompositeFieldPath.
	// +optional
```

Reply:
Good point - I'll fix that.

Reply:
Done!